example/exporter/kube-state-metrics/servicemonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "monitoring.coreos.com/v1alpha1"
+kind: "ServiceMonitor"
+metadata:
+  name: "kube-state-metrics"
+  labels:
+    app: kube-state-metrics
```

Comment:
going with the above comment, labels are used to uniquely identify an object or a number of objects, but as a single `ServiceMonitor` is intended to be used for many `Service`s this should more describe the class of `Service`s discovered. Maybe something in the direction of `k8s-apps: http`.